packages/appstore/src/components/cfds-listing/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import CFDsListing from '../index';
+
+jest.mock('Components/containers/listing-container', () =>
+    jest.fn(({ children }) => <div data-testid='listing-container'>{children}</div>)
+);
+
+describe('CFDsListing', () => {
+    const mock = mockStore({
+        traders_hub: {
```

Comment:
@amina-deriv, there is no need to specify default values which already set in mockStore.
you can just create mockStore({}), and if anything is necessary to be changed from default values, just pass it with objects properties accordingly.
if some fields doesn’t exist in mockStore, please add theme there as a default values

Reply:
Thanks for the suggestion @yauheni-deriv 👍